lib/xml_builder.ex

```diff
@@ -231,16 +232,19 @@ defmodule XmlBuilder do
   defp first_element(element_spec),
     do: element(element_spec)
 
-  defp format_content(children, level) when is_list(children),
-    do: ['\n', Enum.map_join(children, "\n", &format(&1, level))]
+  defp format_content(children, level, options) when is_list(children),
+    do: ['\n', Enum.map_join(children, "\n", &format(&1, level, options))]
 
-  defp format_content(content, _level),
+  defp format_content(content, _level, options),
```

Comment:
Since we're not using `options` in some places, it would be best to use `_` prefix whenever it's unused in that function.

Reply:
oops, yeah, got it